.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
A pedantic nit pick, if in your ~/.git/config you add:

```
[core]
	excludesfile = ~/.gitignore_global
```

And in your `~/.gitignore_global` you add the above, then you exclude it from all your projects

Reply:
I concur.  Operating system and tooling specific excludes should be in a global ignore file rather than the project git ignore file.  Thank you.

Reply:
Nice thank you, fixed.